src/boot/efi/boot.c

```diff
@@ -1993,6 +2009,32 @@ static VOID config_entry_add_linux(
         uefi_call_wrapper(linux_dir->Close, 1, linux_dir);
 }
 
+static BOOLEAN config_entry_add_signing_keys(
```

Comment:
given that the human readable label is now "signature keys", maybe we should rename the function like that too. i.e. `config_entry_add_signature_keys()`.  (other option, abbreviate signature with sig: `config_entry_add_sig_keys()`, but I am not too fond of needless abbreviations)